.github/workflows/ci.yml

```diff
@@ -58,7 +58,7 @@ jobs:
           # Use depth > 1, because sometimes we need to rebuild main and if
           # other commits have landed it will become impossible to rebuild if
           # the checkout is too shallow.
-          fetch-depth: 5
+          fetch-depth: '0'
```

Comment:
I believe this comment still stands - maybe we shouldn't go with depth 5 but 1 or 2 is highly desired

Reply:
`0` means disabling shallow clone i.e. fetching all history. (this is needed for updating the modified timestamps) ref: https://github.com/actions/checkout#checkout-v2

So I'll update the comment.